### PR TITLE
docs: Add info about stability to repo README

### PR DESCRIPTION
**Pull request by ghost**
Add information about AWS Construct Library module stability to repo README for highest possible visibility

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,13 @@ how to use AWS. The AWS Construct Library aims to reduce the complexity and
 glue-logic required when integrating various AWS services to achieve your goals
 on AWS.
 
+Modules in the AWS Construct Library are designated Experimental while we build
+them; experimental modules may have breaking API changes in any relase.  After
```
Comment (shivlaks) [inline README.md:57]:
```suggestion
them; experimental modules may have breaking API changes in any release.  After
```

File: README.md [hunk 2]
```diff
@@ -53,6 +53,13 @@ how to use AWS. The AWS Construct Library aims to reduce the complexity and
 glue-logic required when integrating various AWS services to achieve your goals
 on AWS.
 
+Modules in the AWS Construct Library are designated Experimental while we build
```
Comment (ColeMurray) [inline README.md:56]: Do we have a link to a resource which outlines which modules are experimental and which are stable?

Comment (Jerry-AWS) [reply README.md:56]: I don't know of such a page. I think one could be generated in the API Reference (just a small matter of programming).

Comment (Jerry-AWS) [reply README.md:56]: I'll at least mention that the Overview page of each module in the API Reference has this info.

**Comment by Jerry-AWS** [comment]
Think this is pretty solid now.

**Pull request merged**